### PR TITLE
Validator viewer

### PR DESCRIPTION
**Pull request by AlejandroAdrian**
Signed-off-by: Alejandro <alejandro.adrian@exponentiateam.com>

### CODE REVIEW

File: app/controllers/validators_controller.rb [hunk 1]
```diff
@@ -1,11 +1,17 @@
 class ValidatorsController < ApplicationController
   def new; end
 
+  def show
+    @report = report_or_document
```
Comment (mapreal19) [inline app/controllers/validators_controller.rb:5]: name it record?

File: app/controllers/validators_controller.rb [hunk 2]
```diff
@@ -59,4 +62,57 @@ def read_from_blockchain(transaction_id)
       headers: { "Content-Type" => "application/json" }
     )
   end
+
+  def head_from_address(transaction_id)
+    response = HTTParty.get(
+      "http://195.201.41.35:8008/state?address=#{transaction_id}"
```
Comment (mapreal19) [inline app/controllers/validators_controller.rb:68]: extract the BASE_URL of this address, give a meaningful name, what's the service of the IP address?

File: app/views/validators/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+<div class="flex justify-start m-1.5">
+  <div class="flex flex-row">
+    <div><%= t("validator.action.date") %>: </div>
+    <div><%= @report.date %></div>
+  </div>
+</div>
+<div class="flex justify-start m-1.5">
+  <div class="flex flex-row">
+    <div><%= t("validator.action.block_num") %>: </div>
+    <div><%= @batch_data["data"]["header"]["block_num"] %></div>
```
Comment (mapreal19) [inline app/views/validators/show.html.erb:23]: there is a method `dig` in Ruby that you can use here.

**Review by mapreal19 — CHANGES_REQUESTED** [review]

File: app/controllers/validators_controller.rb (hunk 1 above)
Comment (AlejandroAdrian) [reply app/controllers/validators_controller.rb:5]: Better "validated"?

Comment (mapreal19) [reply app/controllers/validators_controller.rb:5]: nevermind, report is fine

File: app/views/validators/show.html.erb (hunk 3 above)
Comment (AlejandroAdrian) [reply app/views/validators/show.html.erb:23]: Done

**Comment by AlejandroAdrian** [comment]
If it is OK, do not merge. Joaquín will commit the view style

File: app/controllers/validators_controller.rb (hunk 2 above)
Comment (AlejandroAdrian) [reply app/controllers/validators_controller.rb:68]: Done

**Comment by JKinnez** [comment]
I'm going to do merge because I can download de branch

**Pull request merged**